ouroboros-consensus/src/Ouroboros/Consensus/Mempool/API.hs

```diff
@@ -107,6 +107,11 @@ data Mempool m blk idx = Mempool {
       -- transaction/.
     , getTxs :: STM m (Seq (GenTx blk, idx))
 
+      -- | Get all transactions in the mempool, along with their associated
+      -- ticket numbers, which are associated with a ticket number greater
+      -- than the one provided.
+    , getTxsAfter :: idx -> STM m (Seq (GenTx blk, idx))
```

Comment:
Make a note for later that one of the QC properties we want to add is that `getTxsAfter zeroIdx = getTxs` in all circumstances (e.g. so it'd catch the 0 vs 1 bug).